remote/remote_test.go

```diff
@@ -2,19 +2,17 @@ package remote_test
 
 import (
 	"context"
-	"encoding/json"
 	"fmt"
 	"io/ioutil"
 	"math/rand"
-	"net/http"
 	"os"
 	"regexp"
-	"strings"
 	"testing"
 	"time"
 
 	"github.com/docker/docker/client"
-	"github.com/google/go-containerregistry/pkg/authn"
+	ggcrauthn "github.com/google/go-containerregistry/pkg/authn"
+	ggcrv1 "github.com/google/go-containerregistry/pkg/v1"
```

Comment:
why the aliases?

Reply:
I'll revert back. The intent was to obviate the leaky `v1.Platform` uses which will now go away if we remove `WithDefaultPlatform`